addons/default-storage-class/storage-class.yaml

```diff
@@ -44,6 +44,21 @@ volumeBindingMode: WaitForFirstConsumer
 {{ end }}
 
 {{ if eq .Cluster.CloudProviderName "vsphere" }}
+{{ if .Cluster.Features.Has "externalCloudProvider" }}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.beta.kubernetes.io/is-default-class: "true"
+  labels:
+    kubernetes.io/cluster-service: "true"
+  name: vsphere-csi
+provisioner: csi.vsphere.vmware.com
+{{ if .Cluster.VsphereStoragePolicy }}
+parameters:
+  storagepolicyname: {{ .Cluster.VsphereStoragePolicy }}
+{{ end }}
```

Comment:
Does it make sense to create the StorageClass if `.Cluster.VsphereStoragePolicy` is not provided? How that StorageClass would work?

Reply:
It turned out that specifying the `storagePolicy` is not mandatory. If not set, the default one is used.